src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -267,6 +375,57 @@ public IActionResult OrganisationName(string post)
             return View("OrganisationName", organisationVm);
         }
 
+        [Route("TotalEmployees")]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult TotalEmployees(bool? totalEmployeesConfirmation)
+        {
+            var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
+
+            if (!_reportService.CanBeEdited(report))
+                return new RedirectResult(Url.Action("Index", "Home"));
+
+            ViewBag.CurrentPeriod = report?.Period ?? _currentPeriod;
+
+            report.HasMinimumEmployeeHeadcount = totalEmployeesConfirmation.HasValue ? totalEmployeesConfirmation : report.HasMinimumEmployeeHeadcount;
+
+            return View("TotalEmployees", report.HasMinimumEmployeeHeadcount);
+        }
+
+        [Route("PostTotalEmployees")]
```

Comment:
Again, I don't like using the http method in the route name. Should just be TotalEmployees